lib/postgrex/extensions/tsvector.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Postgrex.Extensions.Tsvector do
+  @moduledoc false
+  @behaviour Postgrex.Extension
+
+  def init(_), do: {}
+
+  def matching(_), do: [send: "tsvectorsend"]
```

Comment:
If the format is `:text` should match on `:output` because that specifies the text format, `:send` is for `:binary` format. However we want to use `:binary` format because that composes with "super" extensions (array etc) and is more efficient. It would be very unlikely that we would support a `:text` format extension as a built in at this time.